resources/resinos.robot

```diff
@@ -121,6 +121,25 @@ Run "${image}" on "${application_name}" with delta already enabled
     Wait Until Keyword Succeeds    6x    3s    Device "${tmp_device_uuid}" is offline
     Wait For Process    handle=${handle}    timeout=30s    on_timeout=terminate
 
+Get "${interface}" IP address using socket "${socket}"
+    [Documentation]    Available values for argument ${interface} are:
+    ...                docker0, enp0s3, lo, resin-vpn
+    @{list} =  Create List    docker0    enp0s3    lo    resin-vpn
+    Should Contain    ${list}    ${interface}
```

Comment:
This need not be the case. There could be other interfaces as well when hardware is used. We can drop the constraint of making sure the interface is one of these.